src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss

```
@@ -1,7 +1,41 @@
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/fonts.scss';
 @import '@/ui/style/mixins.scss';
 
 .okp4-dataverse-portal-dataverse-component-page-template-main {
   @include twelve-column-layout;
+  row-gap: 19px;
+
+  .okp4-dataverse-portal-dataverse-back {
+    @include with-theme {
+      display: flex;
```

The positioning of this element in the grid is weak as it implicitly takes the first column in the first row which can give unexpected side effects in the future. 

I would prefer that you give it the whole length of the row.

```suggestion
      @include grid-item($colStart: 1, $colEnd: -1, $rowStart: auto, $rowEnd: auto);
      display: flex;
```